components/topcomponent/TopComponent.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export default function TopComponent({ bookcounter }) {
+  return (
+    <>
+      <StyledHead>
+        <h1>MEDIA DB</h1>
+        {bookcounter ? <span>{bookcounter} books</span> : ``}
+      </StyledHead>
+      <hr />
+    </>
+  );
+}
+
+const StyledHead = styled.div`
```

Comment:
It would be more symantic if you use the `<header>` tag instead.
And rename your styled component to `StyledHeader`